### DISCUSSION TITLE
Can I use Entity Framework ORM to manage CRUD instead of csharp client?

### DISCUSSION THREAD

**Discussion by wgrs** [supabase/supabase#21155] [Questions]
I have fairly complex objects to insert.  1 Parent record has about 8 child records in a one to many relationship. This is a breeze with EF Core but it seems complicated with the CSharp client. But the Csharp client is good at managing auth. I need the best of both worlds. Do I need to run both or should I choose one? My Preference is EF Core but how do I use auth ?

**Comment by ayuksekkaya** [comment]
Was there ever an answer for this? What did you end up doing?
